### PR TITLE
Add more browser tests for file upload question

### PR DESCRIPTION
**Pull request by caitlinshk**
### Description

This PR adds a bunch more browser tests for the file upload question, both required and optional versions.

I'm about to make some changes to how this question works, so I wanted to add some tests beforehand.

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers).
- [x] Created unit and/or browser tests which fail without the change (if possible)
- [x] Performed manual testing (Chrome and Firefox if it includes front-end changes)

### CODE REVIEW

**Review by gwendolyngoetz — CHANGES_REQUESTED** [review]

File: browser-test/src/support/file.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import {Page} from 'playwright'
+
+/** Class for working with the file upload question. */
+export class File {
```
[the remark this answers is not in the document]
Comment (caitlinshk) [reply browser-test/src/support/file.ts]: Updated to `ApplicantFileQuestion`.

**Comment by caitlinshk** [comment]
I updated `bin/run-browser-tests-azure` to not run anything, based on our slack conversation. Let me know if I should disable them in a different way!

File: bin/run-browser-tests-azure [hunk 2]
```diff
@@ -4,10 +4,13 @@
...
 
-
 def get_test_files():
-    test_files = ['src/file.test.ts']  # file upload tests
-    return test_files
+    # Disable Azure browser tests for now. We previously only ran file_upload.test.ts in
+    # Azure (see https://github.com/civiform/civiform/issues/5299), but new file upload
+    # tests aren't passing in Azure. Since there's no CEs planning to deploy with Azure,
+    # it's not worth spending the time to try and fix the new file upload tests and
+    # we'll instead just not run any tests on Azure.
```
Comment (gwendolyngoetz) [inline bin/run-browser-tests-azure:12]: (blocking )Instead of a comment, have this print to the screen so it shows up if someone runs this..

**Review by gwendolyngoetz — CHANGES_REQUESTED** [review]

File: bin/run-browser-tests-azure (hunk 2 above)
Comment (caitlinshk) [reply bin/run-browser-tests-azure]: Updated to print to the screen! I tested locally that running `bin/run-browser-tests-azure` will print this warning then proceed to run the `file_upload` browser tests.

**Pull request merged**